calendar-service/calendar-service-setup.sh

```diff
@@ -7,7 +7,7 @@
 cd ..
 cwd=$(pwd)
 ## Install radicale
-cd $cwd/calendar-service/radicale/linux
+cd "$cwd/calendar-service/radicale/linux"
 tar xvzf Radicale-1.1.1.tar.gz
```

Comment:
the Radicale version should be put in a variable to make it easier when we want to upgrade it later